modules/programs/buf.nix

```diff
@@ -0,0 +1,60 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.buf;
+in {
+  meta.maintainers = [ maintainers.lucperkins ];
+
+  options.programs.buf = {
+    enable = mkEnableOption "The Buf CLI tool";
```

Comment:
Not sure if there is a set convention here. But a lot of packages seem to use `buf, a protocol buffer CLI` style names. 
Then again, a lot don't.

Reply:
I opted for something a bit more descriptive